src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAILY_RENTAL_COST = 40;
+  const DISCOUNT_3_OR_MORE_DAYS = 20;
+  const DISCOUNT_7_OR_MORE_DAYS = 50;
+
+  const totalCost = days * DAILY_RENTAL_COST;
```

Comment:
The variable `totalCost` should be declared with `let` instead of `const` because its value is modified in the conditional statements. Using `let` will allow you to reassign the value when applying discounts.